src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx

```diff
@@ -318,6 +317,19 @@ class Abilities extends CoreAbilities {
         },
         charges: 2,
       },
+      {
+        spell: talents.ABOMINATION_LIMB_TALENT.id,
+        category: SPELL_CATEGORY.COOLDOWNS,
+        cooldown: 120,
+        gcd: {
+          base: 1500,
+        },
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.9,
+        },
+        enabled: false,
```

Comment:
one more tweak

```suggestion
        enabled: combatant.hasTalent(talents.ABOMINATION_LIMB_TALENT),
```

Reply:
```typescript
  /**
   * Whether the spell is enabled (available to the player) and should be
   * displayed. This should only be used for hiding spells that are
   * unavailable, for example due to talents.
   */
  enabled?: boolean;
```

Understood - thank your for pointing that out. I'll keep that in mind during future conversions😅